Cargo.toml

```diff
@@ -17,6 +17,7 @@ path = "src/lib.rs"
 
 [dependencies]
 bitcoin = { version = "0.30.0", optional = true }
+bitcoin-miniscript = { package = "miniscript", version = "10.0" }
```

Comment:
In eb055fa8a5bce506e7752e27426543c54faf8597:

I am tempted to change this from `10.0` to a range from 8.0.1 to 10. Later we will just bump the `10`. Using a range will minimize the amount of grief that we'll have with coordinating new crate versions. (Though arguably we have no need to support miniscript 8 and 9 since those are in the past.)